spec/clipboard.spec.js

```diff
@@ -63,44 +64,69 @@ describe('clipboard.spec: clipboard namespace tests', () => {
         });
         
         it('successfully writes special characters to the clipboard', async () => {
-            let exitCode = runner.run(`
+            runner.run(`
                 await Neutralino.clipboard.writeText('Special characters: @#$%^&*☁☀☊☄');
-                await __close('ok');
+                let clipboardText = await Neutralino.clipboard.readText();
+                await __close(clipboardText);
             `);
-            assert.equal(runner.getOutput(), 'ok');
+            assert.equal(runner.getOutput(), 'Special characters: @#$%^&*☁☀☊☄');
         });
         
         it('successfully writes a large amount of text to the clipboard', async () => {
             let largeText = 'A'.repeat(10000000);
-            let exitCode = runner.run(`
+            runner.run(`
                 await Neutralino.clipboard.writeText('${largeText}');
-                await __close('ok');
+                let clipboardText = await Neutralino.clipboard.readText();
+                await __close(clipboardText);
+            `);
+            assert.equal(runner.getOutput(), largeText);
+        });
+
+        it('successfully writes an empty string to the clipboard', async () => {
+            runner.run(`
+                await Neutralino.clipboard.writeText('');
+                let clipboardText = await Neutralino.clipboard.readText();
+                await __close(clipboardText);
             `);
-            assert.equal(runner.getOutput(), 'ok');
+            assert.equal(runner.getOutput(), '');
+        });
+
+        it('handles concurrent writes to the clipboard', async () => {
+            const text1 = 'Hello';
+            const text2 = 'World';
+            runner.run(`
+                await Promise.all([
+                    Neutralino.clipboard.writeText('${text1}'),
+                    Neutralino.clipboard.writeText('${text2}')
+                ]);
+                let clipboardText = await Neutralino.clipboard.readText();
+                await __close(clipboardText);
+            `);
+            assert.equal(runner.getOutput(), 'World');
```

Comment:
Do we need to assert `Hello` as well since this test is mentioned as "concurrent"? Thanks